templates/lxd/install.html

```diff
@@ -0,0 +1,232 @@
+        <p>
+          <a class="p-button--positive" href="/contact-us" aria-controls="lxd-install-modal">Get in touch</a>
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width">
+  <div class="row--25-75">
+    <div class="col p-section--shallow">
+      <h2>Why choose LXD?</h2>
+    </div>
+    <div class="col">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">1.&ensp;Install LXD as a snap</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <div class="p-section--shallow">
+            <p>To install LXD as a snap, just run:</p>
+            <p><pre>snap install lxd</pre></p>
+          </div>
+        </div>
+      </div>
+      <hr class="p-rule">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">2.&ensp;Configure LXD</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <div class="p-section--shallow">
+            <p>Run the following command and either accept the defaults or choose different options when prompted:</p>
+            <p>
+              <pre>lxd init</pre>
+            </p>
+          </div>
+        </div>
+      </div>
+      <hr class="p-rule">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">3.&ensp;Install the OS you'd like to use in your container or VM</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <div class="p-tabs">
+            <div class="p-tabs__list js-tabbed-content" role="tablist" aria-label="Install the OS">
+              <div class="p-tabs__item">
+                <button class="p-tabs__link" role="tab" aria-selected="true" aria-controls="container-tab"
+                  id="olm">Container</button>
+              </div>
+              <div class="p-tabs__item">
+                <button class="p-tabs__link" role="tab" aria-selected="false" aria-controls="vm-tab" id="vm"
+                  tabindex="-1">VM</button>
+              </div>
+            </div>
+          </div>
+          <div tabindex="0" role="tabpanel" id="container-tab" aria-labelledby="Container">
+            <p>Command:</p>
+            <p>
+            <pre>lxc launch {{"<image_server>:<image_name> <instance_name>"}}</pre>
+            </p>
+            <p>Example:</p>
+            <p>
+            <pre>lxc launch ubuntu:22.04 ubuntu-container</pre>
+            </p>
+            <p>Check the <a href="https://images.linuxcontainers.org/">community image server</a> for other Linux distributions.
+            </p>
+          </div>
+          <div tabindex="0" role="tabpanel" id="vm-tab" aria-hidden="true" aria-labelledby="VM">
+            <p>Command:</p>
+            <p>
+            <pre>lxc launch {{"<image_server>:<image_name> <instance_name>"}} --vm</pre>
+            </p>
+            <p>Example:</p>
```

Comment:
Personally I feel this is a strange way to show an example. It looks like two commands at first.